oxidation/libparsec/crates/types/src/time.rs

```diff
@@ -67,7 +68,7 @@ impl DateTime {
         Self(chrono::Utc.timestamp_opt(t, (us as u32) * 1000).unwrap())
     }
 
-    // Don't implement this as `Into<f64>` to keep it private
+    // Don't implement this as `Into<f64>` to prevent misunderstanding on precision
```

Comment:
```suggestion
    /// Don't implement this as `Into<f64>` to prevent misunderstanding on precision.
```

This information would be useful as a doc-comment

Reply:
I don't think we need that in the docstring: we are documenting why we choose not to provide an implementation, this is something that is not relevant to the user of this structure:

- the user will see it needs to use `from_f64_with_us_precision` and should be happy with it
- if the user is unhappy with `from_f64_with_us_precision` and think there should be a direct conversion with f64, he will head into this file to implement it, then read this message an understand why it wasn't there in the first place